controllers/api.topics.controller.js

```diff
@@ -44,4 +45,26 @@ function getArticleComments(request, response, next) {
     });
 }
 
-module.exports = { getTopics, getArticleById, getArticles, getArticleComments };
+function postComment(request, response, next) {
+  const body = request.body.body;
+  const author = request.body.author;
+  const { article_id } = request.params;
+  if (body.length === 0 || author.length === 0) {
+    next({ status: 400, msg: "400: Bad request" });
+  }
```

Comment:
^^ this should be in the model instead, as the model should ideally deal with all the error rejections :) 

Hint: PSQL might reject so you may have to look into psql error handlers - refer to the notes from last week for this :)